scheduler/test/cook/test/benchmark.clj

```diff
@@ -67,3 +70,25 @@
                                                          sort-task-scored-task-pairs
                                                          sched/sort-jobs-hierarchy-duration))
           nil)))))
+
+(deftest ^:benchmark bench-rank-jobs-state
```

Comment:
From the travis logs, this was 5 ms vs 2 seconds for the other iteration of rank-jobs